cheevos-new/cheevos.c

```diff
@@ -1583,8 +1583,9 @@ static int rcheevos_iterate(rcheevos_coro_t* coro)
             char disc_path[PATH_MAX_LENGTH];
             char* tmp;
 
-            intfstream_read(m3u_stream, buffer, sizeof(buffer));
+            num_read = intfstream_read(m3u_stream, buffer, sizeof(buffer));
             intfstream_close(m3u_stream);
+            buffer[num_read] = '\0';
```

Comment:
this fixes the issue where an m3u without a trailing newline couldn't find the associated cue file.